reloj_checador_team03/config/database.yml

```diff
@@ -24,8 +24,8 @@ default: &default
 development:
   <<: *default
   database: reloj_checador_team03_development
-  username: fego
-  password: kurosu
+  username: mike
+  password: tscom2020
   host: localhost
   port: 5432
 
```

Comment:
Mentioned previously, worth stop tracking this file and replace it with a `.example` one